src/zap-tokens.ts

```diff
@@ -0,0 +1,8 @@
+import { ZapTokenInfo } from './types';
+import { client } from './api';
+import * as common from '@protocolink/common';
+
+export async function getZapTokens(chainId: number): Promise<ZapTokenInfo> {
+  const { data } = await client.get(`/v1/${chainId}/zap-tokens`);
+  return common.classifying(data);
```

Comment:
It could directly return the tokens, which would be more developer-friendly.
[Reference](https://github.com/dinngo/protocolink-js-sdk/blob/master/packages/api/src/api.ts#L34)

Reply:
fixed.